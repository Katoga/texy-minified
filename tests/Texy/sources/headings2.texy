Title
*****

subtitle * 123
==============

/--texysource
**aa**
======

**bb**
------
\--


Title .{toc: jiny toc}
******

last title
----------
