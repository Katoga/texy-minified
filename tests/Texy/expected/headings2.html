<h1 id="toc-title">Title</h1>

<h2 id="toc-subtitle-123">subtitle * 123</h2>

<pre class="html"><code>&lt;h3&gt;&lt;strong&gt;aa&lt;/strong&gt;&lt;/h3&gt;

&lt;h4&gt;&lt;strong&gt;bb&lt;/strong&gt;&lt;/h4&gt;</code></pre>

<h1 id="toc-jiny-toc">Title</h1>

<h3 id="toc-last-title">last title</h3>
